src/consensus/raft/raft.h

```diff
@@ -669,6 +669,11 @@ namespace raft
 
           case kv::DeserialiseSuccess::PASS:
             break;
+
+          case kv::DeserialiseSuccess::PASS_PRE_PREPARE:
```

Comment:
I don't think we need this here as Raft should stay agnostic to other consensuses?

Reply:
Compiler generates warnings if you don't check all the available switch cases. I also think it doesn't hurt to check and fail horribly if this ever happens.

Reply:
We could use `default:`.